Dockerfile

```diff
@@ -0,0 +1,52 @@
+# BUILD:
+# docker build --force-rm=true -t openvpn_exporter .
+
+# RUN:
+# docker run -it -p 9176:9176 -v /path/to/openvpn_server.status:/etc/openvpn_exporter/server.status openvpn_exporter
+
+FROM golang as builder
+
+# Grenoble testing env
+ENV http_proxy="http://web-proxy.gre.hpecorp.net:8080" \
+    https_proxy="http://web-proxy.gre.hpecorp.net:8080"
+
+RUN mkdir /app
+RUN mkdir /go/src/app
+ADD . /go/src/app
+WORKDIR /go/src/app
+
+# Go dep
+RUN go get -u github.com/prometheus/client_golang/prometheus && \
+    go get -u github.com/prometheus/client_golang/prometheus/promhttp
+
+
+# Build a standalone binary
+RUN set -ex && \
+  CGO_ENABLED=0 go build \
+        -tags netgo \
+        -o /app/openvpn_exporter \
+        -v -a \
+        -ldflags '-extldflags "-static"' && \
+  ls
+
+# Create the second stage with a basic image.
+# this will drop any previous
+# stages (defined as `FROM <some_image> as <some_name>`)
+# allowing us to start with a fat build image and end up with
+# a very small runtime image.
+
+FROM busybox
```

Comment:
This container image is now incredibly light-weight. What about using `FROM scratch`?

Reply:
You are right, I used busybox for dev and I forgot to switch back to scratch.

Reply:
Oh yes, I remember now why I finally let busybox. It's to have the "cp" command that I use to copy the compiled binary like mentioned in the doc.
What do you prefer then? I switch to `scratch` anyway?

Reply:
It looks like `docker save` can be used to export files from an image. We could use that in combination with a `FROM scratch` container.

Reply:
It seems that the generated file can be then only used by `docker load` command. Maybe I miss something.

Reply:
Oh, hmmm... Indeed. That's annoying. Well, just stick to Busybox then. :-)

Reply:
Okey ;)